core/src/main/kotlin/org/evomaster/core/problem/rest/service/AbstractRestSampler.kt

```diff
@@ -253,6 +253,11 @@ abstract class AbstractRestSampler : HttpWsSampler<RestIndividual>() {
         if (swagger.paths == null) {
             throw SutProblemException("There is no endpoint definition in the retrieved Swagger file")
         }
+        // Onur: to give the error message for invalid swagger
+        if (swagger.paths.size == 0){
+            throw SutProblemException("The swagger file ${configuration.bbSwaggerUrl} " +
```

Comment:
use `OpenAPI` in output errors and logs. although we might still refer to `swagger` in old source code (which we should had refactored, but it is not a big deal)